docs/os/windows/windows-tools41.md

```diff
@@ -0,0 +1,292 @@
+
+Also, the inter-VM services work as usual -- e.g. to request opening a document or URL in the Windows AppVM from another VM:
+
+~~~
+[user@dom0 ~] $ qvm-open-in-vm my-win-appvm roadmap.pptx
+
+[user@dom0 ~]$ qvm-open-in-vm my-win-appvm https://invisiblethingslab.com
+~~~
+
+... just like in the case of Linux AppVMs. Of course all those operations are governed by central policy engine running in Dom0 -- if the policy doesn't contain explicit rules for the source and/or target AppVM, the user will be asked whether to allow or deny the operation.
+
+Inter-VM file copy and clipboard works for Windows AppVMs the same way as for Linux AppVM (except that we don't provide a command line wrapper, `qvm-copy-to-vm` in Windows VMs) -- to copy files from Windows AppVMs just right-click on the file in Explorer, and choose: Send To-\> Other AppVM.
+
+To simulate Ctrl-Alt-Delete in the HVM (SAS, Secure Attention Sequence), press Ctrl-Alt-Home while having any window of this VM in the foreground.
+
+![windows-seamless-7.png](/attachment/wiki/WindowsAppVms/windows-seamless-7.png)
+
+Changing between seamless and full desktop mode
+-----------------------------------------------
+
+You can switch between seamless and "full desktop" mode for Windows HVMs in their settings in Qubes Manager. The latter is the default.
+
+Using template-based Windows AppVMs
+-----------------------------------
+
+Qubes allows HVM VMs to share a common root filesystem from a select Template VM, just as for Linux AppVMs. This mode is not limited to Windows AppVMs, and can be used for any HVM (e.g. FreeBSD running in a HVM). 
+
+In order to create an HVM TemplateVM, the type "TemplateVM" has to be selected on creating the VM. Then set memory as appropriate, and install the Windows OS (or any other OS) into this template the same way as you would install it into a normal HVM -- please see instructions on [this page](https://www.qubes-os.org/doc/hvm-create/).
+
+If you use this Template as it is, then any HVMs that use it will effectively be DisposableVMs - the User directory will be wiped when the HVM is closed down.
+
+If you want to retain the User directory between reboots, then it would make sense to store the `C:\Users` directory on the 2nd disk which is automatically exposed by Qubes to all HVMs. 
+This 2nd disk is backed by the `private.img` file in the AppVMs' and is not reset upon AppVMs reboot, so the user's directories and profiles would survive the AppVMs reboot, unlike the "root" filesystem which will be reverted to the "golden image" from the Template VM automatically. 
+To facilitate such separation of user profiles, Qubes Windows Tools provide an option to automatically move `C:\Users` directory to the 2nd disk backed by `private.img`. 
+It's a selectable feature of the installer, but currently working only for Windows 10 and 11. 
```

Comment:
Confused with 'but currently working only for Windows 10 and 11'. Move profiles works w/ Windows 7.

Reply:
Oh, that's old text that still needs updating. :-(